src/components/CompareResults/SubtestsResults/SubtestsResultsMain.tsx

```diff
@@ -67,32 +87,71 @@ function SubtestsResultsMain({ view }: SubtestsResultsMainProps) {
     updateRawSearchParams(rawSearchParams);
   };
 
-  if (!results.length) {
+  if (!resolvedResults.length) {
     return (
       <>
         <Container className={styles.container} data-testid='subtests-main'>
-          <header>
-            <SubtestsBreadcrumbs view={view} />
-            <NoResultsFound />
-          </header>
+          {isLoading ? (
+            <>
+              <header>
+                <SubtestsBreadcrumbs view={view} />
+
+                <Alert severity='info' className={styles.title}>
+                  A Perfherder link is available for the same results
+                  {'.'}
+                </Alert>
+                <Stack spacing={1} sx={{ marginBottom: '12px' }}>
+                  <Skeleton
```

Comment:
the color is a little bit dark, I wonder if we can make it somewhat lighter? But no need to think too much